ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml

```diff
@@ -86,6 +86,66 @@ resources:
       cpu: "1"
       memory: 1Gi
 tests:
+- as: aws-ipi-longrun-mco-proxy-fips-p1-f28
+  cron: 6 17 8 * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      FEATURE_SET: TechPreviewNoUpgrade
+      FIPS_ENABLED: "true"
+      OCP_ARCH: arm64
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;~hypershift&;P1&
+      TEST_IMPORTANCE: High
```

Comment:
I think here we need to remove the test-importance here as we selecting P1 test-cases

Reply:
Done